hubitat-teslamate.groovy

```diff
@@ -86,7 +86,7 @@ metadata {
             input(
                 name: "showClimate",
                 type: "bool",
-                title: "Expose climate and temperature details",
+                title: "Expose tire and temperature details",
```

Comment:
Nit-picking, but I would use `climate` and `tire` since temperature is a component of climate.

Reply:
```suggestion
                title: "Expose climate and tire details",
```